### ISSUE TITLE
ImportError: cannot import name '_png' from 'matplotlib

### ISSUE THREAD

**Issue by aegonwolf** [facebook/prophet#2048]
Someone closed this issue https://github.com/facebook/prophet/issues/1691
by giving the downgrading option.
Obviously, this is not a solution for many (including me) because of compatibility issues.

it's still being encountered often judging from SO.

**Comment by init27** [comment]
@aegonwolf Could you please share your setup details? 
I'm running through the examples this week and have installed the library using conda, I didn't run into this error

**Comment by tcuongd** [comment]
Yep it would be great to get your system details and the version of Jupyter you're using. I haven't run into this issue either.

**Comment by JacobHayes** [comment]
For one more tally, I [didn't have this issue](https://github.com/facebook/prophet/issues/1691#issuecomment-1397802599) with the latest versions of both matplotlib and prophet. My hunch is the environments folks are running on may have outdated packages that have since fixed the issue?
